modules/javafx.web/src/main/native/Source/ThirdParty/libxml/src/tree.c

```diff
@@ -5805,131 +5845,263 @@ xmlNodeAddContentLen(xmlNodePtr cur, const xmlChar *content, int len) {
  * NOTE: In contrast to xmlNodeSetContent(), @content is supposed to be
  *       raw text, so unescaped XML special chars are allowed, entity
  *       references are not supported.
+ *
+ * Returns 0 on success, 1 on error, -1 if a memory allocation failed.
  */
-void
+int
 xmlNodeAddContent(xmlNodePtr cur, const xmlChar *content) {
-    int len;
-
-    if (cur == NULL) {
-        return;
-    }
-    if (content == NULL) return;
-    len = xmlStrlen(content);
-    xmlNodeAddContentLen(cur, content, len);
+    return(xmlNodeAddContentLen(cur, content, xmlStrlen(content)));
 }
 
 /**
  * xmlTextMerge:
  * @first:  the first text node
  * @second:  the second text node being merged
  *
- * Merge two text nodes into one
- * Returns the first text node augmented
+ * Merge the second text node into the first. The second node is
+ * unlinked and freed.
```

Comment:
It looks like you missed applying one of the changes from 2.13.7. Replace the above two lines with:

```
 * Merge the second text node into the first. If @first is NULL,
 * @second is returned. Otherwise, the second node is unlinked and
 * freed.
```